freshsales-suite/contact-kyc/app/scripts/contact_entity_menu.js

```diff
@@ -0,0 +1,227 @@
+// ----------------------------
+// Setup
+// ----------------------------
+function $(el) {
+  return document.querySelector(el);
+}
+
+document.onreadystatechange = function () {
+  if (document.readyState === "interactive") {
+    app
+      .initialized()
+      .then(function (client) {
+        client.events.on("app.activated", function () {
+          onAppActivate(client);
+        });
+      })
+      .catch(handleErr);
+  }
+};
+
+async function getContact(client, currentEntityId, iparams) {
+  const url = `${iparams.fcrm_domain}api/contacts/${currentEntityId}`;
+  const options = {
+    headers: {
+      Authorization: `Token token=${iparams.fcrm_api_key}`
+    }
+  }
+  return await client.request.get(url, options);
+}
+
+async function createContactEntity(client, contactDetails) {
+  let contact_entity = await contacts(client);
+  contact_entity.create({
```

Comment:
`await` this. Or return the `Promise` to the caller.

Reply:
awaited the entity create.